content/post/2024-10-01-same-same-but-different-how-advanced-data-science-techniques-help-us-validate-drug-names/index.md

```diff
@@ -0,0 +1,628 @@
+##  [50] "iodine iofetamine (123I)"                      
+##  [51] "isoprenaline"                                  
+##  [52] "istradefylline"                                
+##  [53] "kanamycin"                                     
+##  [54] "lactulose"                                     
+##  [55] "levodopa"                                      
+##  [56] "levonorgestrel"                                
+##  [57] "lincomycin"                                    
+##  [58] "magnesium carbonate"                           
+##  [59] "mecasermin"                                    
+##  [60] "megestrol and estrogen"                        
+##  [61] "menadione"                                     
+##  [62] "methaqualone"                                  
+##  [63] "micafungin"                                    
+##  [64] "moexipril and diuretics"                       
+##  [65] "narcobarbital"                                 
+##  [66] "nebivolol and amlodipine"                      
+##  [67] "nimetazepam"                                   
+##  [68] "odevixibat"                                    
+##  [69] "pegloticase"                                   
+##  [70] "perphenazine"                                  
+##  [71] "pethidine"                                     
+##  [72] "phenylephrine"                                 
+##  [73] "pipotiazine"                                   
+##  [74] "pirprofen"                                     
+##  [75] "plerixafor"                                    
+##  [76] "potassium citrate"                             
+##  [77] "prazosin"                                      
+##  [78] "prednisone"                                    
+##  [79] "remoxipride"                                   
+##  [80] "reteplase"                                     
+##  [81] "rifamycin"                                     
+##  [82] "rivastigmine"                                  
+##  [83] "roxithromycin"                                 
+##  [84] "salsalate"                                     
+##  [85] "sorbitol"                                      
+##  [86] "streptokinase"                                 
+##  [87] "succinimide"                                   
+##  [88] "taurolidine"                                   
+##  [89] "technetium (99mTc) pertechnetate"              
+##  [90] "teneligliptin"                                 
+##  [91] "theophylline, combinations excl. psycholeptics"
+##  [92] "ticarcillin"                                   
+##  [93] "tiemonium iodide and analgesics"               
+##  [94] "timolol, thiazides and other diuretics"        
+##  [95] "tolperisone"                                   
+##  [96] "tramadol"                                      
+##  [97] "tretoquinol"                                   
+##  [98] "trypsin, combinations"                         
+##  [99] "ursodoxicoltaurine"                            
+## [100] "zidovudine"
+```
```

Comment:
Feels a bit redundant, as the structure of the codebook has already been presented earlier. Listing all the names in full doesn’t seem to add much new information and might distract the reader.